content/hi/docs/tasks/extend-kubectl/kubectl-plugins.md

```diff
@@ -0,0 +1,367 @@
+---
+title: kubectl को प्लगइन्स के साथ विस्तारित करें
+reviewers:
+- divya-mohan0209
+- dipesh-rawat
+description: kubectl को प्लगइन बनाकर और इंस्टॉल करके विस्तारित करें।
+content_type: task
+---
+
+<!-- overview -->
+
+यह गाइड आपको [kubectl](/docs/reference/kubectl/kubectl/) के लिए एक्सटेंशन कैसे इंस्टॉल और लिखें, यह दिखाती है।
+मुख्य `kubectl` कमांड्स को कुबेरनेट्स क्लस्टर के साथ इंटरैक्ट करने के लिए आवश्यक बिल्डिंग ब्लॉक्स के रूप में सोचकर,
```

Comment:
Could you ensure the machine translation makes sense to a Hindi reader? The sentence structure here is jumbled.